src/docs/DocsShell/sveldMapper.ts

```diff
@@ -0,0 +1,136 @@
+import type { Component } from './types';
+
+// Mapper: Props
+export function sveldMapperProps(component: Component): any {
+	const { props } = component.sveld;
+	// Filter props with undefined types (exclude getContext)
+	const propsFiltered = props.filter((p: any) => p.type !== undefined);
+	// Return table headings/source
+	return {
+		headings: ['Name', 'Type', 'Value', 'Description'],
+		source: propsFiltered.map((p: any) => {
+			// prettier-ignore
+			return [
+				`<code>${p.name}<?code>`,
+				`<em>${p.type}</em>`,
+				!["''", undefined].includes(p.value) ? p.value : '-',
+				p.description ? p.description : '-'
+			];
+		})
+	};
+}
+
+// Mapper: Slots
+export function sveldMapperSlots(component: Component): any {
+	const { slots } = component.sveld;
+	return {
+		headings: ['Name', 'Default', 'Fallback', 'Description'],
+		source: slots.map((s: any) => {
+			// prettier-ignore
+			return [
+				`<code>${s.name.replaceAll('__','')}</code>`,
```

Comment:
You'll need to bump the lib compiler up to es2021 if you want to use replaceAll on a string.

Reply:
Let's discuss that soon. As long as browser support is good I'm for for it.